environment/uid_windows.go

```diff
@@ -0,0 +1,25 @@
+package environment
+
+import (
+	"os/user"
+	"regexp"
+)
+
+var sidExp = regexp.MustCompile(`([\d\D]+-)(?P<uid>\d+)`)
```

Comment:
If it's being used just inside `uid()`, there is no need to keep it in package scope level. You can declare it inside `uid()`

Reply:
[MustCompile](https://golang.org/pkg/regexp/#MustCompile) is similar [Compile](https://golang.org/pkg/regexp/#Compile), this compile and parses a regular expression

I think it’s not a good idea to compile every time it’s called

Reply:
hum...yeah...it makes sense...never mind...